proxpn

```diff
@@ -136,21 +136,28 @@ TCP_EXIT_NODES[SanJose]=179.48.248.104
 
 UDP_EXIT_NODES[FREE]=196.52.21.65
 UDP_EXIT_NODES[NYC]=196.52.16.2
+UDP_EXIT_NODES[NYC2]=196.52.17.68
 UDP_EXIT_NODES[Miami]=196.52.22.2
+UDP_EXIT_NODES[Miami2]=196.52.23.66
 UDP_EXIT_NODES[Seattle]=196.52.18.2
+UDP_EXIT_NODES[Seattle2]=196.52.19.241
 UDP_EXIT_NODES[Chicago]=50.7.1.67
+UDP_EXIT_NODES[Chicago2]=196.52.21.225
 UDP_EXIT_NODES[Toronto]=162.253.128.67
 UDP_EXIT_NODES[HongKong]=43.251.159.25
-UDP_EXIT_NODES[LA]=64.27.29.74
-UDP_EXIT_NODES[LA2]=64.27.29.70
+UDP_EXIT_NODES[HongKong2]=103.194.43.131
+UDP_EXIT_NODES[HongKong3]=103.194.43.26
+UDP_EXIT_NODES[LA]=188.172.214.196
+#UDP_EXIT_NODES[LA2]=64.27.29.70
 UDP_EXIT_NODES[Netherlands]=213.179.212.2
 UDP_EXIT_NODES[Netherlands2]=213.179.208.146
-UDP_EXIT_NODES[Stockholm]=94.185.84.48
+UDP_EXIT_NODES[Stockholm]=94.185.84.42
```

Comment:
Neither old nor new Stockholm IP seems to work from USA. Could just temporarily be down. New IP is current listed one in xml as of 2016/05/31 00:30 Pacific Time.